index.js

```diff
@@ -91,6 +91,10 @@ const Servicetrade = (options) => {
             request.defaults.headers.Cookie = cookie;
         },
 
+        setBearerToken: (bearerToken) => {
```

Comment:
You should have a unit test of this.  The tests for `setCookie()` are very similar to what you should create for this method, so you can use them as a starting point:  https://github.com/servicetrade/servicetrade-nodejs/blob/38c876ae72dbc8612de2c6d0eac8ff9b226ea7d4/test/index.js#L317